src/controllers/feedController.js

```diff
@@ -0,0 +1,94 @@
+const Feed = require("../models/feed");
+const cloudinary = require("../utils/cloudinary");
+const mongoose = require("mongoose");
+
+exports.createFeed = async (req, res) => {
+  const { createdBy, text } = req.body;
+  try {
+    let result;
+    if (req.file) {
+      result = await cloudinary.uploader.upload(req.file.path);
+    }
+    //instance of post
+    const data = {
+      createdBy: createdBy,
+      text: text,
+      status: "active",
+      imgLink: result?.secure_url || "",
+      cloudinaryId: result?.public_id || "",
+    };
+    let feed = new Feed(data);
+    //saving post
+    await feed.save();
+    res.status(201).json(feed);
+  } catch (error) {
+    res.status(401).send("error" + error);
+  }
+};
+
+exports.getFeed = async (req, res) => {
```

Comment:
Method name seems to return only one Feed. But, returning all the fields in response.